akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -135,6 +135,29 @@ class SubFlow[In, Out, Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Flow[I
   def map[T](f: function.Function[Out, T]): SubFlow[In, T, Mat] =
     new SubFlow(delegate.map(f.apply))
 
+  /**
+   * Similar to [[map]], however does not modify the passed through element, the returned value is ignored.
+   * This is a simplified version of `wireTap(Sink)`, which you may use to wireTap a Sink onto this stream.
```

Comment:
"wireTap a Sink"
copy pasta?

Reply:
was intentional wording actually, "to wire tap" used as a verb... maybe not so natural?

> What does it mean to wire tap?
> an act or instance of tapping telephone or telegraph wires for evidence or other information. ... to listen in on by means of a wiretap: to wiretap a telephone; to wiretap a conversation.